src/video/x11/SDL_x11window.c

```diff
@@ -818,9 +818,9 @@ X11_SetWindowPosition(_THIS, SDL_Window * window)
         X11_XTranslateCoordinates(display, parent, DefaultRootWindow(display),
                                   attrs.x, attrs.y, &x, &y, &childReturn);
 
+        window->x = x;
+        window->y = y;
```

Comment:
The `else if` right below this will break if it's set here, so while we should probably update this more aggressively it'll have to be at a different spot so the stuff below will be reachable.

Reply:
Thank you for taking a look! I force-pushed a fix. Should be good now.